### DISCUSSION TITLE
Multer, Formdata

### DISCUSSION THREAD

**Discussion by Melbourneandrew** [nitrojs/nitro#626] [Q&A] [answered]
How can I use something like (Multer)[https://www.npmjs.com/package/multer] to parse FormData, specifically to handle file uploads? The files only need to remain in memory as after the client uploads them, they will be sent to Cloudflare Images.

Thanks!

**Comment by Twitch0125** [comment, marked as answer]
I realize this is a super late answer, but you can use [readMultipartFormData](https://www.jsdocs.io/package/h3#readMultipartFormData) from h3: 
```ts
    const body = await readMultipartFormData(event);
    const file = body[0]
```

**Reply by atinux** [reply]
Thanks for helping <3
